docs/src/preprocessing/preprocessing.md

```diff
@@ -117,90 +129,82 @@ The reconstructed surface will then look as in the following image.
+v2 = edge_vertices_ids[edge][2]
 
-  vertex_count[v1] += 1
-  vertex_count[v2] -= 1
-```
-```julia
-  # 3D
-  edge1 = face_edges_ids[face][1]
-
-  if edge_vertices_ids[edge1] == (v1, v2)
-      directed_edges[edge1] += 1
-  else
-      directed_edges[edge1] -= 1
-  end
+vertex_count[v1] += 1
+vertex_count[v2] -= 1
 ```
 
-In 2D, a vertex is declared as exterior if `vertex_count(vertex) != 0`.
-The exterior vertices are then connected to one arbitrary
-exterior vertex using appropriately oriented line segments
+In 2D, a vertex is declared as exterior if `vertex_count(vertex) != 0`, so there is not the same amount of edges in this box going into versus out of the vertex.
+To construct the closing surface, the exterior vertices are then connected to one arbitrary
+exterior vertex using appropriately oriented line segments:
 
 ```julia
-  edge = vertex_count[v] > 0 ? (closing_vertex, v) : (v, closing_vertex)
+edge = vertex_count[v] > 0 ? (closing_vertex, v) : (v, closing_vertex)
 ```
 
-In 3D, an edge is declared as exterior if `directed_edges[edge] != 0`
-The exterior edges are triangulated with one exterior vertex $k$ with orientation
-- ``\{i, j, k\}`` if `directed_edges[edge] < 0`
-- ``\{j, i, k\}`` if `directed_edges[edge] > 0`
+The resulting closed surface ``T.S \cup T.\overline{S}`` then has the same number of edges going into and out of each vertex.
 
 #### Incorrect evaluation
 
 If we follow the algorithm, we know that recursion stops if
 
-- The bounding box $T$ is a leaf
-- The query point $\mathbf{p}$ is outside the box
-
-(1): If $\mathbf{p}$ is outside the box, we calculate the winding number with the (red) closure of the box.
-
-(2): If $\mathbf{p}$ is inside the box, we use the (blue) edges distributed to the box.
-
-(3): In this case, it leads to an incorrect evaluation of the winding number.
+- the bounding box $T$ is a leaf or
+- the query point $\mathbf{p}$ is outside the box.
 
 ```@raw html
 <figure>
   <img src="https://github.com/user-attachments/assets/7bae164a-8d5b-4761-9d54-9abf99fca94a" alt="incorrect evaluation"/>
 </figure>
 ```
 
+(1): The query point $\mathbf{p}$ is outside the box, so we calculate the winding number with the (red) closure of the box.
+
+(2): The query point $\mathbf{p}$ is inside the box, so we use the (blue) edges distributed to the box.
+
+(3): In this case, it leads to an incorrect evaluation of the winding number.
+The query point is clearly inside the box, but not inside the reconstructed surface.
+This is because the property $w_{\mathcal{S}}(\mathbf{p}) = - w_{\overline{\mathcal{S}}}(\mathbf{p})$
+only holds when $\mathbf{p}$ is outside of $\mathcal{B}$, which is not the case here.
+
 #### Correct evaluation
-Jacobson et al. (2013) therefore resize the bounding box, since it doesn't matter if the boxes overlap.
+Jacobson et al. (2013) therefore resize the bounding box to fully include the closing surface
+of the neighboring box, since it doesn't matter if the boxes overlap.
```

Comment:
```suggestion
Jacobson et al. (2013) don't mention this problem or provide a solution to it.
We contacted the authors and found that they know about this problem and solve it by resizing the bounding box to fully include the closing surface
of the neighboring box, since it doesn't matter if the boxes overlap.
```
Otherwise, it sounds like they provide a solution in the paper.